src/org/parosproxy/paros/core/scanner/Scanner.java

```diff
@@ -46,15 +46,8 @@
 
 import java.security.InvalidParameterException;
 import java.text.DecimalFormat;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
+import java.util.*;

```

Comment:
Wildcard imports are not usually used, the preference is to just import the required types.